README.md

```diff
@@ -1,71 +1,47 @@
-# Advisor Frontend (AF)
+# Advisor Frontend
 
-[![Build Status](https://travis-ci.com/RedHatInsights/insights-advisor-frontend.svg?branch=master)](https://travis-ci.com/RedHatInsights/insights-advisor-frontend)
-## Getting Started
-There is a [comprehensive quick start guide in the Storybook Documentation](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/welcome/quickStart/DOC.md) to setting up an Insights environment complete with:
-- [Insights-Frontend-Starter-App](https://github.com/RedHatInsights/insights-frontend-starter-app)
-- [Insights Chroming](https://github.com/RedHatInsights/insights-chrome)
-- [Insights Proxy](https://github.com/RedHatInsights/insights-proxy)
+[![Build Status](https://travis-ci.com/RedHatInsights/insights-advisor-frontend.svg?branch=master)](https://app.travis-ci.com/github/RedHatInsights/insights-advisor-frontend)
 
-Note: You will need to set up the Insights environment if you want to develop with the starter app due to the consumption of the chroming service as well as setting up your global/app navigation through the API.
-## Running locally
-1. `npm install`
-2. `npm run start:proxy` or `npm run start:proxy:beta` And enjoy! Thats it! Local proxy is no longer required to run local advisor
-3. https://stage.foo.redhat.com:1337/insights/advisor/ (or whatever enviroment is specified)
-
-Note: The API endpont can be specified by appending 
-`API_ENDOINT=foo_endpoint ` so yah  end up with something like `API_ENDOINT=https://ci.cloud.redhat.com  npm run start` if you wish to specify a ci API.
-
-Additionally, if you would prefer to run it the old way, `npm start` this requires [insights-proxy](https://github.com/RedHatInsights/insights-proxy) to be installed under PROXY_PATH and run (in another terminal) the following command:
+## First time setup
+1. Make sure you have [Node.js](https://nodejs.org/en/) version >= 16 installed
+2. Run [script to patch your `/etc/hosts`](https://github.com/RedHatInsights/insights-proxy/blob/master/scripts/patch-etc-hosts.sh)
+3. Make sure you are using [Red Hat proxy](http://hdn.corp.redhat.com/proxy.pac)
 
-```shell
-SPANDX_CONFIG="./profiles/local-frontend.js" bash $PROXY_PATH/scripts/run.sh
-```
-### Testing
-- Travis is used to test the build for this code.
-    - `npm run test` will run linters and tests
-### Deploying
-The follow six branches are used by IF
-- prod-stable, prod-beta
-- ci-stable, ci-beta
-- qa-stable, qa-beta
-
-A push or merge to master will automatically release to ci-beta and qa-beta
-The same will happen with action against master-stable, it will automatically release to ci-stable and qa-stable
-The prod-beta and prod-stable environments are updated by a deliberate push, (to each branch)
+## Running locally
+1. Install dependencies with `npm install`
+2. Run development server with `npm run start:proxy:beta`
```

Comment:
At the moment theres really no difference between stage and stage-beta with the pathways release, but I still think its a good idea to include that you have the option to run against stage env via `npm run start:proxy` and then specifying that it would be found on its respective env , https://stage.foo.redhat.com:1337/insights/advisor/.

Now that pathways is released it might even be better to update the bash script so that pushing to stage-beta env also pushes to stage

Reply:
Okay, sounds reasonable, I added that, I think it might fall out of fashion soon with more usage of Unleash, but we can remove it afterwards, thanks for review.